tdrs-backend/tdpservice/parsers/schema_defs/header.py

```diff
@@ -21,19 +21,19 @@
             validators.matches('HEADER'),
         ]),
         Field(item="4", name='year', type='number', startIndex=6, endIndex=10, required=True, validators=[
-            validators.between(2000, 2099)
+            validators.isInLimits(2000, 2099)
         ]),
         Field(item="5", name='quarter', type='string', startIndex=10, endIndex=11, required=True, validators=[
             validators.oneOf(['1', '2', '3', '4'])
         ]),
         Field(item="6", name='type', type='string', startIndex=11, endIndex=12, required=True, validators=[
             validators.oneOf(['A', 'C', 'G', 'S'])
         ]),
-        Field(item="1", name='state_fips', type='string', startIndex=12, endIndex=14, required=True, validators=[
-            validators.between(0, 99)
+        Field(item="1", name='state_fips', type='number', startIndex=12, endIndex=14, required=True, validators=[
```

Comment:
can this be changed to `type='string'` that is in the limits `00, 99`? im noticing this show up often in prod for tribes, even though the header is correct. example below:

`HEADER20232C00307TAN1EN`

Reply:
also note that this could be space-filled. ex: `HEADER20232C  307TAN1EN`